make/Makefile.ps1

```diff
@@ -26,6 +26,14 @@ Define-Step -Name 'Testing' -Target 'build' -Body {
 	}
 }
 
+Define-Step -Name 'JS Unit-Testing' -Target 'build' -Body {
+    $NpmPath = Resolve-Path ".\nodejs\npm.cmd" 

```

Comment:
if nodejs comes from nuget package and it is used only for testing, you can fetch it here with command:

$NpmPackagePath = Fetch-Package 'Node.js' '5.3.0'
$NpmPath = "$NpmPackagePath\npm.cmd"